index.html

```diff
@@ -23,12 +23,12 @@
                     <button id="startTime-btn">&#9658;</button>                                                         
                 </div>            
             </div>          
-            <div class="center-box">
+            <div class="center-box"> <!-- Comment -->
                 <div id="display-container">
                     <div id="header-group">                
                         <h1 id="target">000</h1>                 
                     </div>
-                    <div id="table-container">
+                    <div id="table-container"> <!-- Comment -->
```

Comment:
I've forgotten what I wanted to say here.  Oh well, must not have been important :smile: